jbmc/src/java_bytecode/java_bytecode_convert_method.cpp

```diff
@@ -647,13 +647,13 @@ static irep_idt get_if_cmp_operator(const irep_idt &stmt)
   throw "unhandled java comparison instruction";
 }
 
-static member_exprt to_member(
-  const exprt &pointer, const exprt &fieldref, const namespacet &ns)
+static member_exprt
+to_member(const exprt &pointer, const exprt &fieldref, const namespacet &ns)
 {
   struct_tag_typet class_type(fieldref.get(ID_class));
 
-  const exprt typed_pointer = typecast_exprt::conditional_cast(
-    pointer, java_reference_type(class_type));
+  const exprt typed_pointer =
```

Comment:
This commit contains format fixups that ought to be squashed into a previous commit